engine/runtime/src/main/java/org/enso/interpreter/runtime/builtin/Builtins.java

```diff
@@ -132,44 +122,43 @@ public Builtins(Context context) {
 
     createPolyglot(language);
 
-    scope.registerMethod(io, "println", PrintlnNode.makeFunction(language, scope));
-    scope.registerMethod(io, "print_err", PrintErrNode.makeFunction(language));
-    scope.registerMethod(io, "readln", ReadlnNode.makeFunction(language));
+    scope.registerMethod(io, "println", PrintlnMethodGen.makeFunction(language));
```

Comment:
I really want to see all of these put into separate files like the `Bool` file above. A separate file per atom type, I think?